lua/github-theme/theme.lua

```diff
@@ -453,6 +453,25 @@ theme.setup = function(cfg)
     ALEWarningSign = { fg = c.warning },
     ALEErrorSign = { fg = c.error },
 
+    -- Barbar
+    BufferCurrent = { fg = c.fg },
```

Comment:
ℹ️  This also had `bg = c.bg`, but adding that made the icons have a light background when using the github dark theme. I found that other themes use `guibg=none` so I removed bg.